src/Components/AddToCart.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import CartState from '../recoils/CartState';
+import { useRecoilState } from 'recoil';
+import { useState } from 'react';
+import '../Styles/Cart.css';
+const update = (items, idx, item) => {
+    if (item.inCartQuantity === 0) {
+        return [...items.slice(0, idx), ...items.slice(idx + 1)]
+    } else {
+        return [...items.slice(0, idx), item, ...items.slice(idx + 1)]
+    }
+}
+
+function AddToCart(props) {
```

Comment:
use consistent function notation or arrow function syntax for functional components.